lib/core/ui/dashboard/home/home_calendar.dart

```diff
@@ -1,24 +1,44 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:live_score/core/constants/dashboard_constants.dart';
+import 'package:live_score/core/extensions/date_time_extension.dart';
 import 'package:live_score/core/theme/app_theme.dart';
 import 'package:live_score/core/ui/dashboard/home/home_date_element.dart';
-import 'package:live_score/core/utils/date_util.dart';
 import 'package:live_score/features/dashboard/cubit/dashboard_cubit.dart';
 
 class HomeCalendar extends StatelessWidget {
   const HomeCalendar({super.key});
 
+  void scrollLeft(ScrollController scrollController) {
+    scrollController.animateTo(
+      scrollController.offset - DashboardConstants.calendarScrollOffset,
+      duration: const Duration(
+        milliseconds: DashboardConstants.calendarScrollDuration,
+      ),
+      curve: Curves.easeInOut,
+    );
+  }
+
+  void scrollRight(ScrollController scrollController) {
+    scrollController.animateTo(
+      scrollController.offset + DashboardConstants.calendarScrollOffset,
+      duration: const Duration(
+        milliseconds: DashboardConstants.calendarScrollDuration,
+      ),
+      curve: Curves.easeInOut,
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
+    ScrollController scrollController = ScrollController();
+    final DateTime now = DateTime.now();
```

Comment:
This line shouldn't be here, this means that a new DateTime object is created every time the widget is rebuilt. I'd move this back to `addPostFrameCallback`.

Reply:
ok, add a TODO comment here for now if it works, e.g 'refactor later', I'll approve your changes and we can come back to this later.